sandbox/indeed-scraping/indeed-scraper.py

```diff
@@ -0,0 +1,209 @@
+#!/usr/bin/python
+
+import sys
+from bs4 import BeautifulSoup
+from selenium import webdriver
+from selenium.common.exceptions import TimeoutException
+import time
+import requests
+import random
+import pandas as pd
+import boto3
+import botocore
+import json
+import logging
+
+# Locations in Bay Area, CA where there many jobs for each search term (for ex: Software Engineer), so, let use a smaller radius
+# than the default of 25miles. With this smaller radius, there is lesser chance of crossing 1000. While paginating through next links,
+# indeed.com only allows a max of 1000. 
+LOCATIONS_BAY_AREA_CA = {
+             "San+Jose%2C+CA":10, 
+             "Los+Altos%2C+CA":10,
+             "Sunnyvale%2C+CA":10,
+             "Mountain+View%2C+CA":10,
+             "Palo+Alto%2C+CA":10,
+             "Redwood+City%2C+CA":10,
+             "San+Mateo%2C+CA":10,
+             "San+Francisco%2C+CA":10,
+             "Milpitas%2C+CA":10,
+             "Fremont%2C+CA":10,
+             "Dublin%2C+CA":10, 
+             "Foster+City%2C+CA":10
+             }
+      
+# other locations in CA       
+LOCATIONS_OTHER_CA = {
+                      "Los+Angeles%2C+CA":50,
+                      "Sacramento%2C+CA":50
+                      }
+
+# other locations in US
+LOCATIONS_OTHER_STATES = {
```

Comment:
an improvement in the future (definitely not necessary now) would be to point the script at these location JSONs using `argparse` so this could be used as a command-line executable. Might make it easier to use as a general-purpose tool

(to be clear though, I'm not recommending you do this. Just thinking out loud)

Reply:
makes sense